sar/README.md

```diff
@@ -178,8 +181,8 @@ None of the fields is required.
 | Field Name             | Data Type | XML Tag                | Description                                                  |
 | ---------------------- | --------- | ---------------------- | ------------------------------------------------------------ |
 | lut_applied            | string    | `lutApplied`           |                                                              |
-| range_look_bandwidth   | \[number] | `RangeLookBandwidth`   | Range Look Bandwidth per swath, convert to GHz if required.  |
-| azimuth_look_bandwidth | \[number] | `AzimuthLookBandwidth` | Azimuth Look Bandwidth per swath, convert to GHz if required. |
+| range_look_bandwidth   | \[number] | `RangeLookBandwidth`   | Range Look Bandwidth per swath in hertz (Hz).                |
```

Comment:
To align with the SAR extension, the value was intentionally in GHz. Please revert or give reasons for the change.

Reply:
I see the reasoning why this was aligned with existing fields of the SAR extension and I guess also why GHz was chosen. However, the more I think of this, the stronger my opinion grows in using hertz here and also in the SAR extension. Hence, my comment that I'm going to open an issue there.  But, I can of course revert these changes for now.

To reduce ambiguity in the metadata, the usage of SI base units or SI derived units (e.g., Hz) should be a priority. That's why I also changed `orbit_mean_altitude` to meters instead of kilometers. Furthermore, this is also a question of aligning the extension 1:1 to the CARD4L metadata document, where SI / SI derived units are being used.

Reply:
I think it will be hard to change this to Hz in the SAR extension as otherwise it would get rather confusing with some documents containing Hz and some GHz. You'd need to look up version numbers to distinguish them, which is undesirable. So the only way to change this is a completely new field.

I'm not an SI unit expert, but aren't km and GHz also SI-derived units?

So for the sake of consistency of units in the metadata files, I'd go everywhere with GHz to align with the SAR extension and with m for compliance with SI base units. It would be even more confusing if in a single file you have sometimes km (or GHz) and sometimes m (Hz).

Reply:
>I think it will be hard to change this to Hz in the SAR extension as otherwise it would get rather confusing with some documents containing Hz and some GHz. You'd need to look up version numbers to distinguish them, which is undesirable. So the only way to change this is a completely new field.

I understand. On the other hand it will be even more difficult to change in the future once both extensions have a stable maturity status, are published and are being widely used (e.g. in data products such as the one we're developing). 

>I'm not an SI unit expert, but aren't km and GHz also SI-derived units?

I'm also not an expert, but these are just SI base (m) and SI derived (Hz) units with a prefix. In the case of STAC JSON, where no unit is defined with the data itself, it leads to ambiguity. That's why I think it's best to stick with SI base and derived units and use scientific notation. For example:  `5.405e09` for central frequency. 
 
>So for the sake of consistency of units in the metadata files, I'd go everywhere with GHz to align with the SAR extension and with m for compliance with SI base units. It would be even more confusing if in a single file you have sometimes GHz and sometimes Hz (same applies for m/km, so +1 for the meter change).

I'd be okay with this for now and agree that it's good to be consistent at least. I'd be very much interested in other people's opinions on this though!

Reply:
47d7797